src/commands/cli/schemas/compare.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as path from 'path';
+import * as os from 'os';
+import { FlagsConfig, SfdxCommand } from '@salesforce/command';
+import { fs, Messages, SfdxError } from '@salesforce/core';
```

Comment:
[`fs` is deprecated. Should use `fs/promises` instead.](https://github.com/forcedotcom/sfdx-core/blob/main/src/util/fs.ts#L28)
```suggestion
import * as fs from 'fs/promises';
import { Messages, SfdxError } from '@salesforce/core';
```